frontend/plugins/insurance_ui/src/modules/insurance/components/shared/entityColumns.tsx

```
@@ -30,11 +31,14 @@ export const createEntityMoreColumn = <T,>(
 // Generic name column factory
 export const createNameColumn = <T,>(
   icon: TablerIcon,
-  label: string = 'Name',
+  label: string = 'name',
 ): ColumnDef<T> => ({
   id: 'name',
   accessorKey: 'name',
-  header: () => <RecordTable.InlineHead icon={icon} label={label} />,
+  header: () => {
+    const { t } = useTranslation('insurance');
```

ESLint

[error] React Hook "useTranslation" is called in function "header" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)